synthnn/core/accelerated_musical_network.py

```
@@ -6,9 +6,11 @@
 """
 
 import numpy as np
-from typing import Dict, List, Optional, Any, Tuple
+import warnings
+from typing import Dict, Optional, Any
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)